### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by antoine-boue**

### CODE REVIEW

**Comment by antoine-boue** [comment]
Ah I forgot to write the instructions lol.

So I guess if I remember well there are two main things left which are the left nav and the characteristic table at the end.

I guess that left nav thing that follow the scrolling might be really annoying for me so if you want to look at it, might be nice. 
There is also another thing that I didn't finish which is the slider CSS, still not really optimize for any picture size, the little ones too. But most of the css is already done and JavaScript is done too.

I'll do that table, add the shop section which might not be so easy considering the page layout, and few other things still to fix that I will do probably.

Let's talk more about it in details later.

File: views/pages/product.php [hunk 1]
```diff
@@ -1,6 +1,278 @@
...
+          </div>
+        <?php endforeach ?>
+      </div>
+    </div>
+  </div>
 </div>
-<?php include('../shared/foot.php'); ?>
+
+<script>
+  const slides = document.getElementsByClassName("product-slider__image");
```
Comment (windresistance) [inline views/pages/product.php:245]: Another way you can do this is to inject the php array directly into the js:
```const slideImages = <?= json_encode($slide_images) ?>;```

File: views/pages/product.php [hunk 2]
```diff
@@ -1,6 +1,278 @@
...
+            <img class="product-slider__image" src=<?= $imgPath . $image ?>>
+          <?php endforeach ?>
+        </div>
+        <div class="product-slider__button" onClick="nextSlide()">
+          <div class="product-slider__icon product-slider__icon--right"></div>
+        </div>
+      </div>
+      <div class="sub-slides">
+        <?php foreach ($slide_images as $x => $image) : ?>
+          <div class="sub-slide" onClick="changeSlide(<?= $x ?>)">
```
Comment (windresistance) [inline views/pages/product.php:185]: clever :)

File: views/pages/product.php (hunk 1 above)
Comment (windresistance) [reply views/pages/product.php:245]: then you won't need to query the DOM and use ```Array.from(slides)``` below

**Review by windresistance — APPROVED** [review]

**Pull request merged**